storage/client.go

```diff
@@ -83,7 +83,7 @@ type storageClient interface {
 	ComposeObject(ctx context.Context, req *composeObjectRequest, opts ...storageOption) (*ObjectAttrs, error)
 	RewriteObject(ctx context.Context, req *rewriteObjectRequest, opts ...storageOption) (*rewriteObjectResponse, error)
 
-	OpenReader(ctx context.Context, r *Reader, opts ...storageOption) error
+	OpenReader(ctx context.Context, params *openReaderParams, opts ...storageOption) (*Reader, error)
```

Comment:
This is really `NewRangeReader` rather than `OpenReader` right? It seems to correspond to the constructor method and you wouldn't call it again if the reader needs to be reopened.

Reply:
Yeah, I think you are right. I will rename it accordingly.

Reply:
Done!